src/presenter/film-presenter.js

```diff
@@ -91,7 +97,7 @@ export default class FilmPresenter {
     }
   };
 
-  #openPopup = () => {
+  #openPopup = async () => {
```

Comment:
Загружать комменты нужно при открытии попапа, поэтому весь код по инициализации нужно перенести в этот метод (не просто добавить, а именно перенести), ты уже сделала его async, что правильно. После загрузки комментов останется только создать компонент попапа, передав в него загруженные данные
```suggestion
  #openPopup = async () => {
    this.#commentsModel = new CommentsModel(new CommentsApiService(END_POINT, AUTHORIZATION));
    await this.#commentsModel.init(this.#movie.id);
    const comments = this.#commentsModel.comments;
    this.#popupComponent = new PopupFilmView({...this.#movie, comments});
```